app/matplotlib.ipynb

```diff
@@ -194,21 +209,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
-    "\n",
-    "# write your code here"
+    "plt.pie(market_share, labels=labels, autopct='%1.1f%%', startangle=90)\n",
+    "plt.title(\"Market Share of Products\")"
```

Comment:
You need to add `plt.show()` at the end of this cell to display the pie plot as per the task requirements.